R/test-elements.R

```diff
@@ -1060,8 +1068,8 @@ checkbox_page <-
            subprompt = "",
            labels = NULL,
            trigger_button_text = "Continue",
-           failed_validation_message = "Choose at least one answer!",
-           force_answer = FALSE,
+           failed_validation_message = c(one = "Choose at least one answer!", all = "Please check all boxes."),
+           force_answer = c( "no", "one", "all", "first"),
```

Comment:
What is the point of 'all' and 'first'?

Reply:
All these different consent pages... came directly out of customer requests.

Reply:
Ah, a consent page. That makes sense.